lib/MusicBrainz/Server/Form/Collection.pm

```diff
@@ -78,4 +78,32 @@ sub validate_type_id {
     }
 }
 
+sub validate_collaborators {
+    my $self = shift;
+
+    my @collaborators = $self->field('collaborators')->fields;
+    my $is_valid = 1;
+    for my $collaborator (@collaborators) {
+        my $id_field = $collaborator->field('id');
+        my $name_field = $collaborator->field('name');
+        my $editor = $self->ctx->model('Editor')->get_by_name($name_field->value);
```

Comment:
Would be better to move this line inside the block below, above `if (defined $editor) {`, since it's not used outside that block and we probably don't want to run this if `$name_field->value` is actually `undef`.